app/features/dynamo/tools.py

```diff
@@ -1,20 +1,20 @@
 from langchain_community.document_loaders import YoutubeLoader
 from langchain_text_splitters import RecursiveCharacterTextSplitter
 from langchain.prompts import PromptTemplate
-from langchain_google_genai import GoogleGenerativeAI
+from langchain_google_vertexai import VertexAI
 from langchain_core.output_parsers import JsonOutputParser
 from langchain.chains.summarize import load_summarize_chain
 from langchain_core.pydantic_v1 import BaseModel, Field
-from app.api.error_utilities import VideoTranscriptError
+from api.error_utilities import VideoTranscriptError
 from fastapi import HTTPException
-from app.services.logger import setup_logger
+from services.logger import setup_logger
 import os
 
 
 logger = setup_logger(__name__)
 
 # AI Model
-model = GoogleGenerativeAI(model="gemini-1.0-pro")
+model = VertexAI(model="gemini-1.0-pro")
```

Comment:
Why is `langchain_google_genai.GoogleGenerativeAI` being changed to `langchain_google_vertexai.VertexAI` in another feature, while you're still using `langchain_google_genai.GoogleGenerativeAI` in your syllabus generator?